src/core_modules/capture-core/components/WidgetTwoEventWorkspace/WidgetTwoEventWorkspace.component.js

```diff
@@ -0,0 +1,64 @@
+// @flow
+import React, { useMemo } from 'react';
+import { spacers } from '@dhis2/ui';
+import { FlatList } from 'capture-ui';
+import { pipe } from 'capture-core-utils';
+import withStyles from '@material-ui/core/styles/withStyles';
+import { convertClientToView, convertServerToClient } from '../../converters';
+import type { RenderFoundation } from '../../metaData';
+import { getDataEntryDetails, Placements } from './utils/getDataEntryDetails';
+
+type Props = {
+    linkedEvent: any,
+    dataValues: any,
+    formFoundation: RenderFoundation,
+    classes: {
+        container: string,
+    },
+}
+
+const styles = {
+    container: {
+        padding: `${spacers.dp8} ${spacers.dp16}`,
+        display: 'flex',
+        flexDirection: 'column',
+        gap: spacers.dp8,
+    },
+};
+
+const convertFn = pipe(convertServerToClient, convertClientToView);
+
+const WidgetTwoEventWorkspacePlain = ({ linkedEvent, dataValues, formFoundation, classes }: Props) => {
+    const dataEntryValues = useMemo(() => getDataEntryDetails(
+        linkedEvent,
+        formFoundation,
+    ), [linkedEvent, formFoundation]);
+
+    const listValues = useMemo(() => {
+        const elements = formFoundation.getElements();
+        const convertedValues = formFoundation.convertAndGroupBySection(dataValues, convertFn);
```

Comment:
The dataElements of type `IMAGE` and `FILE` are not displayed. Can you check why the `PreviewImage` component and the file name URL are missing? Thanks!
<img width="824" alt="Screenshot 2024-07-30 at 16 18 00" src="https://github.com/user-attachments/assets/73fe8be4-a6a1-4339-8e7a-9c0d8dec54e9">